modules/ROOT/pages/6.4.2-release-notes.adoc

```diff
@@ -76,6 +76,16 @@ include::partial$misc/admon-releasenotes-for-stable.adoc[]
 
 // {productname} 6.4.2 also includes the following bug fixes:
 
+=== Tab navigation no longer incorrectly stops at menu buttons within toolbar groups.
+
+In previous versions of {productname}, having a menu button in the “more” floating toolbar, would cause issues with the navigation behaviour.
+
+As a consquence, Tabbing would stop at the menu button, rather than stopping at the toolbar groups **first** button.
```

Comment:
```suggestion
As a consequence, Tabbing stopped at the menu button, rather than stopping at the toolbar group’s first button.
```
Copy-edits and typos.